wren-ai-service/src/providers/llm/openai.py

```diff
@@ -293,44 +129,34 @@ def _verify_api_key(api_key: str, api_base: str) -> None:
 
         logger.info(f"Initializing OpenAILLM provider with API base: {api_base}")
         # TODO: currently only OpenAI api key can be verified
-        if api_base == OPENAI_API_BASE:
+        if api_base == LLM_OPENAI_API_BASE:
             _verify_api_key(api_key.resolve_value(), api_base)
-            logger.info(f"Using OpenAI Generation Model: {generation_model}")
+            logger.info(f"Using OpenAI LLM: {generation_model}")
         else:
-            logger.info(
-                f"Using OpenAI API-compatible Generation Model: {generation_model}"
-            )
+            logger.info(f"Using OpenAI API-compatible LLM: {generation_model}")
         self._api_key = api_key
         self._api_base = api_base
-        self._embedding_model = embedding_model
-        self._embedding_model_dim = embedding_model_dim
         self._generation_model = generation_model
 
     def get_generator(
         self,
-        model_kwargs: Optional[Dict[str, Any]] = GENERATION_MODEL_KWARGS,
+        model_kwargs: Dict[str, Any] = orjson.loads(
+            os.getenv("GENERATION_MODEL_KWARGS", "{}")
+        )
+        or GENERATION_MODEL_KWARGS,
```

Comment:
we assumed a default value when the env var is not exist thus I don't think the OR operator works here.

Reply:
ignore the above comment, I am wrong, it works. 😅

```python
>>> {} or {'f':1}
{'f': 1}
```